store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -597,4 +606,17 @@ private String toTypeName(short type) {
         }
         return "Unrecognized type " + type;
     }
-}
\ No newline at end of file
+
+    /**
+     * Returns the storage type name for the given volume info type.
+     */
+    private String extractStorageType(VolumeInfo volInfo) {
+        String result = "";
```

Comment:
Here, we don't need to initialize with blank. Initialize with null only

Reply:
ok fixed